tests/test_exceptions.py

```diff
@@ -104,6 +112,89 @@ def test_valid_TraversedPartialPath_nibbles(valid_nibbles):
         ((0, -1), ValueError),
     ),
 )
-def test_invalid_TraversedPartialPath_nibbles(invalid_nibbles, exception):
+def test_invalid_TraversedPartialPath_traversed_nibbles(invalid_nibbles, exception):
     with pytest.raises(exception):
-        TraversedPartialPath(invalid_nibbles, annotate_node(b''))
+        TraversedPartialPath(invalid_nibbles, annotate_node(b''), (1,))
+
+
+@pytest.mark.parametrize(
+    'valid_nibbles',
+    (
+        (0, 0, 0),
+        (0xf, ) * 128,  # no length limit on the nibbles
+    ),
+)
+@pytest.mark.parametrize('key_encoding', (compute_extension_key, compute_leaf_key))
+def test_valid_TraversedPartialPath_untraversed_nibbles(valid_nibbles, key_encoding):
```

Comment:
odd casing for the method name but I know it's on purpose and if you happen to like that sort of thing I won't argue against it :)